### PR TITLE
Fabo/use backend testnet

### PR DESCRIPTION
**Pull request by faboweb**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -113,14 +113,11 @@ jobs:
             exit 0
 
   testE2e:
-    docker:
-      # Primary container image where all steps run.
-     - image: *docker-browsers
+    mashine: true
```
Comment (jbibla) [inline .circleci/config.yml]: i don't know what this is exactly, but `machine` probably

**Comment by faboweb** [comment]
Requires #3043

**Comment by faboweb** [comment]
@colw I propose we merge this as the working setup for e2e

**Comment by jbibla** [comment]
ready to be merged or needs to wait for #3043?

File: src/vuex/modules/connection.js [hunk 2]
```diff
@@ -1,11 +1,10 @@
 import config from "src/../config"
 
-export default function () {
+export default function() {
   const state = {
     stopConnecting: false,
     connected: true, // TODO do connection test
     network: config.network, // network id to reference network capabilities stored in Hasura
-    connectionAttempts: 0,
```
Comment (jbibla) [inline src/vuex/modules/connection.js]: we don't use this anymore?

File: tests/e2e/send.spec.js [hunk 3]
```diff
@@ -26,7 +26,7 @@ module.exports = {
     browser.url(browser.launch_url + "/#/transactions")
     await waitForText(
       browser,
-      ".tx:first-child .tx__content__caption",
+      ".tx:nth-child(2) .tx__content__caption",
```
Comment (jbibla) [inline tests/e2e/send.spec.js]: is this a reliable strategy?

File: tests/unit/specs/scripts/url.spec.js [hunk 4]
```diff
@@ -4,13 +4,14 @@ describe(`URL functions`, () => {
   it(`gets query params`, () => {
     const windowMock = {
       location: {
-        search: `?stargate=x&rpc=y`
+        search: `?stargate=x&network=y&graphql=z`
```
Comment (jbibla) [inline tests/unit/specs/scripts/url.spec.js]: do we still need stargate?

File: src/vuex/modules/connection.js (hunk 2 above)
Comment (faboweb) [reply src/vuex/modules/connection.js]: it's not in our codebase anymore

File: tests/e2e/send.spec.js (hunk 3 above)
Comment (faboweb) [reply tests/e2e/send.spec.js]: do you have a better strategy?

File: tests/unit/specs/scripts/url.spec.js (hunk 4 above)
Comment (faboweb) [reply tests/unit/specs/scripts/url.spec.js]: yes it's still used in `cosmos-api` to simulate and broadcast

**Comment by faboweb** [comment]
Merging to not have this open for too long. Merges with feature branch and needs to be perfected

**Pull request merged**